test/translator/report_luns_test.cc

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "lib/translator/report_luns.h"
+#include "lib/translator/common.h"
+
+#include "absl/base/casts.h"
+#include "gtest/gtest.h"
+
+#include <byteswap.h>
+#include <netinet/in.h>
```

Comment:
do we need another kernel check here?

Reply:
The Bazel testing suite runs in user space & thus these tests are not runnable in the kernel, so I omitted the kernel check